docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Quickwit"
+---
+
+## Configuring Quickwit Backend
+
+The only required field is the Quickwit endpoint. This endpoint must be a valid URL for your Quickwit cluster.
+The following fields are optional:
+
+- **Traces Index** - The name of the index where traces will be stored. Defaults to `trace_index`.
+- **Logs Index** - The name of the index where logs will be stored. Defaults to `log_index`.
```

Comment:
these fields are not in the yaml at the moment. should they be added to the yaml and go code, or removed from the docs?

Reply:
I think removed from the doc, because I'm not sure Quickwit support overriding those default index's name for OTEL (but if @fmassot can confirm or says otherwise). I remove from the doc for now (it's coming from the elasticsearch's doc).

Reply:
It’s possible to override the default index name with a custom header.

Reply:
Thanks @fmassot I've found it on the doc:

https://quickwit.io/docs/log-management/send-logs/using-otel-collector
https://quickwit.io/docs/distributed-tracing/otel-service#sending-spans-in-your-own-index

The headers are: `qw-otel-logs-index and `qw-otel-traces-index`

But I don't know yet how we can configure those headers (maybe @blumamir have an idea, thanks in advance). We could do it in another PR though, not mandatory to make it work